Library/Homebrew/cmd/update-report.rb

```diff
@@ -376,16 +376,16 @@ def install_from_api_message
     api_auto_update_secs = Homebrew::EnvConfig.api_auto_update_secs.to_i
     api_auto_update_secs_default = Homebrew::EnvConfig::ENVS.fetch(:HOMEBREW_API_AUTO_UPDATE_SECS).fetch(:default)
     auto_update_secs_set = api_auto_update_secs.positive? && api_auto_update_secs != api_auto_update_secs_default
-    if !Homebrew::EnvConfig.no_auto_update? && !Homebrew::EnvConfig.no_install_from_api? && !auto_update_secs_set
+    no_auto_update_set = Homebrew::EnvConfig.no_auto_update? && !ENV["CI"]
```

Comment:
```suggestion
    no_auto_update_set = Homebrew::EnvConfig.no_auto_update? && (ENV["HOMEBREW_GITHUB_HOSTED_RUNNER"] || ENV["GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"])
```

Reply:
Is the suggestion supposed to be
```suggestion
    no_auto_update_set = Homebrew::EnvConfig.no_auto_update? && !(ENV["HOMEBREW_GITHUB_HOSTED_RUNNER"] || ENV["GITHUB_ACTIONS_HOMEBREW_SELF_HOSTED"])
```
? Mind the `!`.

Reply:
Yes, sorry. Whatever the correct version is hopefully you get the gist.